doc/user/bgp.rst

```diff
@@ -2693,6 +2693,11 @@ The following commands can be used in route maps:
    happen only when BGP updates have completely same communities value
    specified in the community list.
 
+.. clicmd:: match community-limit (1-1024)
```

Comment:
Any reason why this is limited to 1k? And why we can't use 0?

Reply:
mmm, 0 would mean we drop BGP updates which have any communities. I think this can be added.